front/src/common/Map/Layers/blankStyle.ts

```diff
@@ -66,7 +66,7 @@ export const useMapBlankStyle = (): MapProps['mapStyle'] => {
       name: 'Blank',
       sources: {},
       sprite,
-      glyphs: 'https://static.osm.osrd.fr/fonts/{fontstack}/{range}.pbf',
+      glyphs: `${window.location.origin}${baseURL}/fonts/{fontstack}/{range}.pbf`,
```

Comment:
Hm, I don't think we should be using `window.location.origin` here, `baseURL` alone should be enough. `baseURL` is taken from `PROXY_EDITOAST` from the environment and can contain a host.

That said, this is copy-pasted from elsewhere so orthogonal to this PR.